### PR TITLE
Add schedulingGate Label verify

### PR DESCRIPTION
**Pull request by lwan-wanglin**
Add schedulingGate Label verify

### CODE REVIEW

File: pkg/e2e/operator/pod_placement_config_test.go [hunk 1]
```diff
@@ -210,3 +217,29 @@ func verifyPodNodeAffinity(ns *corev1.Namespace, labelKey string, labelInValue s
...
+	Expect(err).NotTo(HaveOccurred())
+	Eventually(func(g Gomega) {
+		pods := &corev1.PodList{}
+		err := client.List(ctx, pods, &runtimeclient.ListOptions{
+			Namespace:     ns.Name,
+			LabelSelector: labelSelector,
+		})
+		g.Expect(err).NotTo(HaveOccurred())
+		g.Expect(pods.Items).NotTo(BeEmpty())
+		for i := 0; i < len(pods.Items); i++ {
```
Comment (aleskandro) [inline pkg/e2e/operator/pod_placement_config_test.go]: I'm not sure, but this `for` loop can be ginkgo-native as in:

```go
g.Expect(pods.Items).Should(HaveEach(WithTransform(func(p Pod) {
  return p.Labels
}, matcher))
```

Where `matcher` replaces `executor` and can be:

```go
func PresentKVMatcher(k, v string) GomegaMatcher {
  return SatifyAll(Not(BeEmpty()), HaveKeyWithValue(k, v)
}
func AbsentKVMatcher(k string) GomegaMatcher {
  return NotTo(HaveKey(k))
}

```


  https://onsi.github.io/gomega/#haveeachelement-interface

Comment (lwan-wanglin) [reply pkg/e2e/operator/pod_placement_config_test.go]: thanks ,will try this way.

Comment (lwan-wanglin) [reply pkg/e2e/operator/pod_placement_config_test.go]: fixed

**Comment by aleskandro** [comment]
This needs rebase after #102 merges, right?

**Comment by lwan-wanglin** [comment]
> This needs rebase after #102 merges, right?

right, thanks, rebased.

**Review by aleskandro — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**